libs/sdk-backend-tiger/src/backend/features/feature.ts

```diff
@@ -128,6 +128,13 @@ export function mapFeatures(features: FeaturesMap): Partial<ITigerFeatureFlags>
             "BOOLEAN",
             FeatureFlagsValues.enableSingleSelectionFilter,
         ),
+        ...loadFeature(
+            features,
+            TigerFeaturesNames.EnableCompositeGrain,
+            "enableCompositeGrain",
+            "BOOLEAN",
+            FeatureFlagsValues.enableCompositeGrain,
```

Comment:
We also need define this FF in (ISettings) that is parto of model 

https://github.com/gooddata/gooddata-ui-sdk/blob/70c7c43de4d481fcca8d75b640c7e5598d94d09b/libs/sdk-model/src/settings/index.ts#L270

In case this FF will be used in Dashboard component its necessary define selector and setup default value

Reply:
Example of FF selector here 
https://github.com/gooddata/gooddata-ui-sdk/blob/70c7c43de4d481fcca8d75b640c7e5598d94d09b/libs/sdk-ui-dashboard/src/model/store/config/configSelectors.ts#L304